CMakeLists.txt

```diff
@@ -17,11 +17,18 @@ include(GNUInstallDirs)
 
 include(CMakePackageConfigHelpers)
 
+if(MSVC AND NOT DEFINED NOFORTRAN)
+  set(NOFORTRAN ON)
+  set(TIMER "NONE")
+endif()
+
```

Comment:
IIUC, this effectively reverts ed8c028f7fc30cc2327410b8051c99f8eb1f58cb.

Should this line be removed from the patch?